packages/gasket-plugin-https/index.js

```diff
@@ -37,8 +37,9 @@ async function start(gasket) {
   if (http2) configOpts.http2 = http2;
 
   const serverOpts = await gasket.execWaterfall('createServers', configOpts);
-  const { healthcheck, ...terminusDefaults } = await gasket.execWaterfall('terminus', {
+  const { healthcheck, healthcheckHtmlRoute, ...terminusDefaults } = await gasket.execWaterfall('terminus', {
     healthcheck: '/healthcheck',
+    healthcheckHtmlRoute: '/healthcheck.html',
```

Comment:
Glad we can allow for multiple routes. To simplify the config signature, and keep it flexible, could we default `healthcheck` to an array of route names, but which supports a single string if set?
```
const routes = Array.isArray(healthcheck) ? healthcheck : [healthcheck]
```

Reply:
> Glad we can allow for multiple routes. To simplify the config signature, and keep it flexible, could we default `healthcheck` to an array of route names, but which supports a single string if set?
> 
> ```
> const routes = Array.isArray(healthcheck) ? healthcheck : [healthcheck]
> ```

I love this idea as well